src/index.ts

```diff
@@ -302,6 +302,8 @@ export default function solidPlugin(options: Partial<Options> = {}): Plugin {
         typeof extension === 'string' ? extension : extension[0],
       );
 
+      id = id.replace(/\?.+$/, '');
```

Comment:
I think `+` should ideally change to `*`, in case of empty query.

Reply:
solved in 
- #191